javaslang/src/main/java/javaslang/collection/Array.java

```diff
@@ -26,10 +26,10 @@
 
     private static final Array<?> EMPTY = new Array<>(new Object[0]);
 
-    private final Object[] back;
+    private final Object[] delegate;
```

Comment:
All the operations are delegated to this entity, hence the rename.
`back` can have too many meanings.